service/selectors/entities/users.js

```diff
@@ -110,3 +110,13 @@ export const getAutocompleteUsersInCurrentChannel = createSelector(
         return autocompleteUsersInChannel[currentChannelId] || {};
     }
 );
+
+export function searchedProfiles(state) {
```

Comment:
Maybe make this into an actual selector? It's not taking any additional props other than the state so it should be doable.